docs/manual.rst

```diff
@@ -2095,20 +2095,20 @@ efficient than accumulating with :meth:`union`.
     >>> m = MultiPolygon(polygons)
     >>> m.area
     7.6845438018375516
-    >>> cascaded_union(m).area
+    >>> unary_union(m).area
     6.6103013551167971
 
+.. function:: shapely.ops.cascaded_union(geoms)
+
+  Returns a representation of the union of the given geometric objects.
+
   .. note::
 
-     In 1.2.16 :func:`shapely.ops.cascaded_union` is superceded by
+     Since 1.2.16 :func:`shapely.ops.cascaded_union` is superceded by
      :func:`shapely.ops.unary_union` if GEOS 3.2+ is used. The unary union
```

Comment:
For GEOS 3.3+ this note doesn't seem to be fully correct, as `cascaded_union` is from then on aliased to `unary_union` instead of using the older implementation

Reply:
True. I think I'll let it go for now.